app/renderer/components/common/browserButton.js

```diff
@@ -88,7 +88,7 @@ const styles = StyleSheet.create({
     cursor: 'pointer',
     lineHeight: 1.25,
     width: 'auto',
-    height: 'auto',
+    height: '32px',
```

Comment:
The reason why setting this value is to force the same height with the buttons on the bookmark hanger and the notification bar. They slightly (~2px) differ by setting it to `auto`.

Known issue: with this change the buttons on Brave payments gets broken. Setting their heights to `37px` would fix the issue.